test/models/artefact_test.rb

```diff
@@ -231,6 +231,19 @@ class ArtefactTest < ActiveSupport::TestCase
     end
   end
 
+  should "validate redirect_url" do
+    artefact = FactoryGirl.create(:artefact)
+
+    artefact.redirect_url = "foobar"
+    refute artefact.valid?
+
+    artefact.redirect_url = "/foobar"
+    assert artefact.valid?
+
+    artefact.redirect_url = "http://foo.bar/"
+    assert artefact.valid?
```

Comment:
As long as we're explicit in the apps using this that only GDS editors can perform these redirects I'm okay with this.